source/git.js

```diff
@@ -9,6 +9,7 @@ var inherits = require('util').inherits;
 var addressParser = require('./address-parser');
 var GitTask = require('./git-task');
 var isWindows = /^win/.test(process.platform);
+var Promise = require("bluebird");
```

Comment:
Minor: use single quotes `'` instead of double quotes `"` as [here](https://github.com/FredrikNoren/ungit/pull/602/files#diff-c195f9bd38b09dd6f6f59b618f867367R10)